frontend/src/app/containers/CoursesList.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+export default() => {
+    return (
+        <>
+        </>
+    )
+};
```

Comment:
you can delete this as it doesnt do anything

Reply:
This one was prepared for the courses list. I will delete it for the moment then.